docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md

```diff
@@ -405,3 +405,142 @@ Algorithm       Hash
 ---------       ----                                                              ----
 SHA256          F46597C0C63A1EEFB200D40EDF654E52F10C3D5D21565886AD603FABAF8D39FB  E:\test_file.txt
 ```
+
```

Comment:
@PrzemyslawBanasiak13 should we improve the test matrix for any of Dasharo's products? As I know, this feature is dedicated only to MSI Z690-A

Reply:
Updated test matrices for Novacustom platforms